src/clients/api/queries/getIsolatedPools/formatOutput/index.ts

```diff
@@ -238,13 +238,14 @@ const formatToPools = ({
     // Calculate userPercentOfLimit for each asset
     const formattedAssets: Asset[] = assets.map(asset => ({
       ...asset,
-      userPercentOfLimit: pool.userBorrowLimitCents
-        ? new BigNumber(asset.userBorrowBalanceCents)
-            .times(100)
-            .div(pool.userBorrowLimitCents)
-            .dp(2)
-            .toNumber()
-        : 0,
+      userPercentOfLimit:
+        asset.userBorrowBalanceCents?.isGreaterThan(0) && pool.userBorrowLimitCents
```

Comment:
We need to check that `pool.userBorrowLimitCents` is positive in order to avoid a division by 0:
```suggestion
        asset.userBorrowBalanceCents?.isGreaterThan(0) && pool.userBorrowLimitCents?.isGreaterThan(0)
```